Looking in indexes: https://pypi.tuna.tsinghua.edu.cn/simple
Collecting moviepy==2.dev2
  Using cached moviepy-2.0.0.dev2-py3-none-any.whl
Collecting gevent==0.12.2
  Using cached https://pypi.tuna.tsinghua.edu.cn/packages/f9/9d/34801e8a25656bbb02f4e7bf361ac8b69b5bdb08e1a3a758042085d587b6/gevent-0.12.2.tar.gz (206 kB)
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'error'
  error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [7 lines of output]
      Traceback (most recent call last):
        File "<string>", line 36, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "/tmp/pip-install-j9vyme98/gevent_3de10e76278844cebd7031a8426799df/setup.py", line 81
          print 'Linking %s to %s' % (path_to_build_core_so, path_to_core_so)
                                 ^
      SyntaxError: invalid syntax
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
